pkg/rotator/rotator.go

```diff
@@ -539,7 +539,7 @@ func ValidCert(caCert, cert, key []byte, dnsName string, at time.Time) (bool, er
 
 // controller code for making sure the CA cert on the
 // webhooks don't get clobbered
-
+/*
```

Comment:
It looks like you've inlined all of this code into `addController` - what's the reason for that? `addController` is already fairly long. Either way, if you _do_ inline this into `addController`, shouldn't this code all be deleted, and not commented out? Also, can we make this part of a separate change, i.e. not part of the upgrade to controller-runtime 0.7.0?

Reply:
Forgot to properly clean this up will do that right now!
Indeed instead of inlining I can just pass a function. The reason for the change is simply that the function now a function instead of an interface, this is an upstream change in controller runtime

Reply:
Changed it please let me know if you think the current version is better

Reply:
lgtm, thanks!